docs/kafka/consumer-configuration-kafka/README.adoc

```diff
@@ -139,10 +139,13 @@ sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule require
 == Adding a consumer to a consumer group
 
 [role="_abstract"]
+Consumer groups are created by specifying a consumer group ID in the user's client application, and not from the CLI or web console.
```

Comment:
Two things I suggest changing here:

 - Write in the active voice, not the passive voice.
 - Don't refer to "the user" in the third person. Our documentation is directed **to** the user. 
 
So, something like:
`To create a consumer group, you specify a consumer group ID in a client application. You do *not* create consumer groups in the {product-kafka} web console, or using the CLI.`